lib/l10n/intl_en.arb

```diff
@@ -255,4 +255,11 @@
       "numberOfItems": {}
     }
   },
+  "items_selected": "{numberOfItems} items selected",
```

Comment:
small note: `Intl` support plurals. Please check in details: https://pub.dev/packages/intl

Reply:
Done thx